android/app/src/main/java/app/organicmaps/car/CarAppSession.java

```diff
@@ -198,7 +199,7 @@ public void onMyPositionModeChanged(int newMode)
 
   public void onCompassUpdated(double north)
   {
-    Map.onCompassUpdated(north, false);
+    Map.onCompassUpdated(north, true);
```

Comment:
What does it change?

Reply:
That's hard to explain.
The arrow is connected to compass.
When you start the app from AA, the arrow will be freezed and won't rotate until the first compass forced redraw.